### PR TITLE
Add marketplace guide pages

### PR DESCRIPTION
**Pull request by NicolasMahe**
This is still in WIP as I'm waiting for new release of website to integrate screenshot!

### CODE REVIEW

File: guide/marketplace/wallet.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Wallet
+
+The wallet contains the Ethereum accounts that you will be able to use to interact with the Marketplace.
+
+The wallet holds the private keys of the Ethereum accounts and never expose it.
+
+::: tip Backup everything!
```
Comment (NicolasMahe) [inline guide/marketplace/wallet.md]: @antho1404 can I use other color for the this box? I would like a red or orange color

Comment (antho1404) [reply guide/marketplace/wallet.md]: `::: warning` or `::: error` should work (not sure about the error, it might be alert)

**Comment by antho1404** [comment]
https://marketplace.mesg.com/ is updated you can now insert screenshot if you need to

File: guide/marketplace/deploy.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+__INSERT SCREENSHOT OF MARKETPLACE__
+
+
+## Deploy a Service
+
+If the service is free (the service has at least one offer with a price of 0 MESG Token), you can directly deploy it! Otherwise, you first need to [purchase an access](#purchase-an-access).
+
+To deploy a service, click on the deploy button, copy the command and execute it in your terminal.
+For the [`email-sendgrid`](https://marketplace.mesg.com/services/email-sendgrid) service, it looks like:
+```
```
Comment (antho1404) [inline guide/marketplace/deploy.md]:
```suggestion
```bash
```

File: guide/marketplace/wallet.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Wallet
+
+The wallet contains the Ethereum accounts you can use to interact with the Marketplace.
+
+The wallet holds the private keys of the Ethereum accounts and never exposes them.
+
+::: tip Back up everything!
```
Comment (antho1404) [inline guide/marketplace/wallet.md]:
```suggestion
::: warning Back up everything!
```

**Pull request merged**